ansible/roles/containerd/README.adoc

```diff
@@ -0,0 +1,52 @@
+== Role: Kubernetes Cluster
+
+Install containerd on host.
+
+=== Description
+
+This role will install a containerd on a linux machine.
+
+
+=== Requirements
+
+TODO
+
+=== Role Variables
+
+[cols="1,1,1,4"]
+|===
+| Name | Type | Value | Description
+
+| state | string | [present,absent] 
+a|
+
+  * `present` will install docker 
```

Comment:
Sorry but I'm confused. Is the goal of this role to install : a) `containerd` only or b) `containerd` and `docker`. If the answer is `b)`, then the description of the role should mention what we exactly can install

Reply:
From what I red in the [Install Docker Engine on CentOS](https://docs.docker.com/engine/install/centos/) Docker document, the Docker Engine also needs containerd so it is included in the installation. But containerd doesn't need docker.